src/System.Collections/src/System/Collections/Generic/SortedSet.cs

```diff
@@ -333,8 +328,9 @@ object ICollection.SyncRoot
         #region Subclass helpers
 
         //virtual function for subclass that needs to update count
-        internal virtual void VersionCheck() { }
-
+        internal virtual void VersionCheck()
+        {
+        }
```

Comment:
Yeah, I agree on this one. The style guide sort of implies that braces should go on their own line, but it's not really specifically referring to situations like this.